src/patternfly/components/Table/table.scss

```diff
@@ -455,6 +455,14 @@
     --pf-c-table-cell--PaddingLeft: var(--pf-c-table__action--PaddingLeft);
   }
 
+  .pf-c-table__check {
```

Comment:
All of the table cells should be top-aligned. The table vertical alignment should match the data list component. Looks like you could probably just add `vertical-align: baseline;` to `.pf-c-table tr > *`

Reply:
Yes this looks good now!